src/main/java/de/epiceric/shopchest/listeners/HologramUpdateListener.java

```diff
@@ -20,8 +20,17 @@ public HologramUpdateListener(ShopChest plugin) {
 
     @EventHandler
```

Comment:
And as you added the following to the PlayerMoveEvent in the ShopItemListener, you can also add it here.
```java
@EventHandler(priority = EventPriority.HIGH, ignoreCancelled = true)
```